core/tests/protractor_utils/ExplorationEditorImprovementsTab.js

```diff
@@ -28,12 +29,24 @@ var ExplorationEditorImprovementsTab = function() {
   };
 
   this.expectHealthyExploration = async function() {
+    await waitFor.visibilityOf(
+      explorationHealth,
+      'Exploration health taking too long to appear'
```

Comment:
```suggestion
      'Exploration health is taking too long to appear'
```

Ditto below in this file.

Reply:
Done